src/ssl.c

```diff
@@ -866,18 +866,27 @@ int wolfSSL_SetMinEccKey_Sz(WOLFSSL* ssl, short keySz)
     return SSL_SUCCESS;
 }
 
-int wolfSSL_GetEccMyKey(WOLFSSL* ssl, struct ecc_key** key)
+int wolfSSL_GetEccKey(WOLFSSL* ssl, struct ecc_key** key)
```

Comment:
This function needs some documentation.  Get which key?  Static, peer, ephemeral.  Looks like it handles a few cases.